master/getting-started/kubernetes/tutorials/app-layer-policy/index.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Application layer policy tutorial
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/tutorials/app-layer-policy/'
+---
+
+The included demo sets up a microservices application, then demonstrates the use of application
+layer policy to mitigate some common threats.
+
+## Pre-requisites
+
+To create a Kubernetes cluster which supports application layer policy, follow
+one of our [getting started guides]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes),
+and [enable application layer policy]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/app-layer-policy).
+
+You will also need the [calicoctl]({{site.baseurl}}/{{page.version}}/usage/calicoctl/install) command line tool.
+
+### Install the demo application
+
+We will use a simple microservice application to demonstrate Calico application layer policy.  The 
+[YAO Bank] application creates a customer-facing web application, a microservice that serves up
+account summaries, and an [etcd] database.
+
+```bash
+kubectl apply -f \
+{{site.url}}/{{page.version}}/getting-started/kubernetes/tutorials/app-layer-policy/manifests/10-yaobank.yaml
+```
+
+> **Note**: You can also
+> [view the manifest in your browser](manifests/10-yaobank.yaml){:target="_blank"}.
+{: .alert .alert-info}
+
+When the demo application has come up, you will see 3 pods.
+
+    kubectl get pods
```

Comment:
We are trying to separate the commands from the responses with some words, to be more explicit about which is which and also to facilitate copy/paste.